docs/docs/icicle/primitives/ntt.md

```diff
@@ -232,9 +146,11 @@ Mixed Radix can reduce the number of stages required to compute for large inputs
 
 ### Which algorithm should I choose ?
 
-Radix 2 is faster for small NTTs. A small NTT would be around logN = 16 and batch size 1. Its also more suited for inputs which are power of 2 (e.g., 256, 512, 1024). Radix 2 won't necessarily perform better for smaller `logn` with larger batches.
+Both work only on inputs of power of 2 (e.g., 256, 512, 1024).
```

Comment:
I know this is true for mixed radix, is it true for radix 2 too? Yuval, Dimma, or Vitaly should know.

Reply:
I asked them just to make sure

Reply:
verified that this is indeed correct.